Changelog for v47.4.0 (#19604)
ckeditor/ckeditor5: CHANGELOG.md

#### Experimental table cell type support

We are introducing an experimental `tableCellTypeSupport` flag that enables changing table cell types between data and header cells (`th`). This feature provides more flexibility when working with complex table structures. Read more about how to enable it in the [update guide](https://ckeditor.com/docs/ckeditor5/latest/updating/guides/update-to-47.html#update-to-ckeditor-5-v4740).

#### Hidden table borders visualization

We are introducing a new `config.table.showHiddenBorders` configuration option (enabled by default) that helps editors work with tables that have hidden borders. When the editor detects inline `border:none` or `border-style:none` declarations on table and cell elements, it renders dashed helper borders in the editing view. This makes it easier to see the table structure while editing without affecting the output data. For strict WYSIWYG scenarios where you want the editing view to match the output exactly, this visualization can be disabled.

#### Email compatibility improvements

> We added an [inline-styles transformation](https://ckeditor.com/docs/ckeditor5/latest/framework/deep-dive/conversion/attributes.html) that improves the appearance of resized inline images in classic Outlook clients. Additionally, the new optional `useFigureToTableFallback` flag in the email inline-styles transformers can replace figures (block images) with tables to improve alignment and width handling in older email clients with limited CSS support.

Reviewer: This one, @Witoso ?

https://ckeditor.com/docs/ckeditor5/latest/features/email-editing/email.html#email-specific-style-transformations

> We added an [inline-styles transformation](https://ckeditor.com/docs/ckeditor5/latest/framework/deep-dive/conversion/attributes.html) that improves the appearance of resized inline images in classic Outlook clients. Additionally, the new optional `useFigureToTableFallback` flag in the email inline-styles transformers can replace figures (block images) with tables to improve alignment and width handling in older email clients with limited CSS support.

Reviewer 2: ```suggestion
We improved the [email styles transformation](https://ckeditor.com/docs/ckeditor5/latest/features/email-editing/email.html#email-specific-style-transformations) with better appearance of resized inline images in classic Outlook clients. Additionally, the new optional `useFigureToTableFallback` flag in the email styles transformers can replace `figure` (block images) with tables to improve alignment and width handling in older email clients with limited CSS support.
```

> We added an [inline-styles transformation](https://ckeditor.com/docs/ckeditor5/latest/framework/deep-dive/conversion/attributes.html) that improves the appearance of resized inline images in classic Outlook clients. Additionally, the new optional `useFigureToTableFallback` flag in the email inline-styles transformers can replace figures (block images) with tables to improve alignment and width handling in older email clients with limited CSS support.

Reviewer 2: yes, something like this: https://github.com/ckeditor/ckeditor5/pull/19604/changes#r2681389916
